projects/angular/src/forms/input/input.ts

```diff
@@ -28,5 +28,14 @@ export class ClrInput extends WrappedFormControl<ClrInputContainer> {
     el: ElementRef<HTMLInputElement>
   ) {
     super(vcr, ClrInputContainer, injector, control, renderer, el);
+    if (this.el.nativeElement.getAttribute('type') === 'number') {
+      console.warn(`Warning: Inputs of type "number" should utilize the number-input component for proper handling.\n
```

Comment:
The datagrid number filter triggers this warning. That should be fixed.

http://localhost:4200/demo/datagrid/string-filtering?cds-theme=light